charts/tidb-operator/templates/admission/admission-webhook-deployment.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if eq .Values.admissionWebhook.create true }}
+{{- if .Values.admissionWebhook.create }}
```

Comment:
why we have to change the criteria

Reply:
Cause `{{- if eq .Values.admissionWebhook.create true }}` didn't work well when I configure the setting by `helm upgrade <name> <path> --set key=value` which would casue and comparing type error.

Reply:
This is should be our helm BP